### ISSUE TITLE
automatic builds need to learn about autorelease and autochangelog

### ISSUE THREAD

**Issue by mattdm** [fedora-infra/the-new-hotness#348]
Looks like the thing which auto-bumps and makes new test builds doesn't know to leave release and changelog alone when the new automatic macros are in use. See  https://bugzilla.redhat.com/attachment.cgi?id=1824987&action=diff for example.

![image](https://user-images.githubusercontent.com/123367/135768446-25b34a5d-551f-43f0-aa49-bdf30575883d.png)

**Comment by Zlopez** [comment]
We are using `rpmdev-bumpspec` utility to update the `.spec` file. So if this tool is updated to work with `autorelease` macro then we only need to update it to latest version.

**Comment by Zlopez** [comment]
This is now solved. Following is what I did:
1. Found out the [rpmdevtools update](https://pagure.io/rpmdevtools/c/5b662f1f9c4a3f0082684d59f0950099ec81caa5?branch=main) was released in version 9.4, but on the OpenShift pod is only 8.10
2. The pod was running F32 which only has rpmdevtools 8.10
3. Updated the pod in OpenShift to F34, which contains latest version of rpmdevtools
4. the-new-hotness is now running of F34 in both production and staging

As a bonus I updated the Vagrant dev environment for the-new-hotness #349
